### PR TITLE
change validations slightly

### PR DESCRIPTION
**Pull request by pjshumphreys**
https://trello.com/c/OdDLpQ7r
https://trello.com/c/nE7O8N8i

This work also includes feedback from https://trello.com/c/UsiGWFJU:
 Column P - the total should be "No" unless all workplaces are "Yes", in which case the total is "Yes".
And column "N" - relevant to relevantSocialCareQualification when "no" is displayed in red

### CODE REVIEW

**Comment by pjshumphreys** [comment]
[LA User Report - Last update date should go black if updated in window](https://trello.com/c/nE7O8N8i/329-la-user-report-last-update-date-should-go-black-if-updated-in-window)

**Comment by pjshumphreys** [comment]
[LA User Report - Last update date should go black if updated in window](https://trello.com/c/nE7O8N8i/329-la-user-report-last-update-date-should-go-black-if-updated-in-window)

**Comment by pjshumphreys** [comment]
[LA User Report - Last update date should go black if updated in window](https://trello.com/c/nE7O8N8i/329-la-user-report-last-update-date-should-go-black-if-updated-in-window)

**Comment by pjshumphreys** [comment]
[LA User Report - Last update date should go black if updated in window](https://trello.com/c/nE7O8N8i/329-la-user-report-last-update-date-should-go-black-if-updated-in-window)

**Comment by pjshumphreys** [comment]
[LA User Report - Col Q heading should say Total Staff](https://trello.com/c/OdDLpQ7r/330-la-user-report-col-q-heading-should-say-total-staff)

File: server/routes/reports/localAuthorityReport/user.js [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
 const schema = 'http://schemas.openxmlformats.org/spreadsheetml/2006/main';
 const isNumberRegex = /^[0-9]+(\.[0-9]+)?$/;
 
-//XML DOM manipulation helper functions
+// XML DOM manipulation helper functions
 const { DOMParser, XMLSerializer } = new (require('jsdom').JSDOM)().window;
 
 const parseXML = fileContent =>
-  (new DOMParser()).parseFromString(fileContent.toString('utf8'), "application/xml");
+  (new DOMParser()).parseFromString(fileContent.toString('utf8'), 'application/xml');
```
Comment (wozzer72) [inline server/routes/reports/localAuthorityReport/user.js]: why are you changing quotes on a defect?

File: server/routes/reports/localAuthorityReport/user.js [hunk 2]
```diff
@@ -86,7 +85,7 @@ const identifyLocalAuthority = async postcode => {
   // lookup primary authority by trying to resolve on specific postcode code
   const cssrResults = await models.pcodedata.findOne({
     where: {
-      postcode,
+      postcode
```
Comment (wozzer72) [inline server/routes/reports/localAuthorityReport/user.js]: why are you adding a trailing comma here on a defect?

File: server/routes/reports/localAuthorityReport/user.js [hunk 3]
```diff
@@ -100,16 +99,15 @@ const identifyLocalAuthority = async postcode => {
   if (cssrResults && cssrResults.postcode === postcode &&
     cssrResults.theAuthority && cssrResults.theAuthority.id &&
     Number.isInteger(cssrResults.theAuthority.id)) {
-
-    return cssrResults.theAuthority.name
+    return cssrResults.theAuthority.name;
```
Comment (wozzer72) [inline server/routes/reports/localAuthorityReport/user.js]: Why are you deleting a blank line on defect?

File: server/routes/reports/localAuthorityReport/user.js (hunk 1 above)
Comment (pjshumphreys) [reply server/routes/reports/localAuthorityReport/user.js]: This is an automatic linting fix from `semistandard --fix`

File: server/routes/reports/localAuthorityReport/user.js (hunk 2 above)
Comment (pjshumphreys) [reply server/routes/reports/localAuthorityReport/user.js]: It was unnecessary and was removed automatically

File: server/routes/reports/localAuthorityReport/user.js (hunk 3 above)
Comment (pjshumphreys) [reply server/routes/reports/localAuthorityReport/user.js]: I've run the code through a linter to introduce some formatting consistency to the code I wrote

**Comment by omarCreativeDev** [comment]
@pjshumphreys what are you using for formatting just out of interest? FE is using prettifier...

**Pull request merged**